.github/workflows/ci.yaml

```diff
@@ -252,14 +252,14 @@ jobs:
         if: needs.changes.outputs.test_full_suite == 'true'
         run: |
           . venv/bin/activate
-          pre-commit run --hook-stage manual black --all-files --show-diff-on-failure
+          pre-commit run --verbose --hook-stage manual black --all-files --show-diff-on-failure
       - name: Run black (partially)
         if: needs.changes.outputs.test_full_suite == 'false'
         shell: bash
         run: |
           . venv/bin/activate
           shopt -s globstar
-          pre-commit run --hook-stage manual black --files {homeassistant,tests}/components/${{ needs.changes.outputs.integrations_glob }}/**/* --show-diff-on-failure
+          pre-commit run --verbose --hook-stage manual black --files {homeassistant,tests}/components/${{ needs.changes.outputs.integrations_glob }}/**/* --show-diff-on-failure
```

Comment:
Verbose only adds additional output to `pre-commit` when the job passes.
Therefore, in this situation, it is not adding value in here for that matter. I suggest reverting these `--verbose` changes.